docs/configuring-playbook-jitsi.md

```diff
@@ -53,20 +63,35 @@ matrix_jitsi_prosody_auth_internal_accounts:
     password: "another-password"
 ```
 
-**Caution:** Accounts added here and subsquently removed will not be automatically removed from the Prosody server until user account cleaning is integrated into the playbook.
+**Caution:** Accounts added here and subsequently removed will not be automatically removed from the Prosody server until user account cleaning is integrated into the playbook.
 
 **If you get an error** like this: "Error: Account creation/modification not supported.", it's likely that you had previously installed Jitsi without auth/guest support. In such a case, you should look into [Rebuilding your Jitsi installation](#rebuilding-your-jitsi-installation).
 
+### Authenticate using Matrix OpenID (Auth-Type 'matrix')
 
-### (Optional) LDAP authentication
+**Attention: Probably breaks jitsi in federated rooms and does not allow sharing conference links with guests.**
 
-The default authentication mode of Jitsi is `internal`, however LDAP is also supported. An example LDAP configuration could be:
+Using this authentication type require a [Matrix User Verification Service](https://github.com/matrix-org/matrix-user-verification-service).
+By default, this playbook creates and configures a user-verification-service to run locally, see [configuring-user-verification-service](configuring-playbook-user-verification-service.md).
+
+To enable set this configuration at host level:
+
+```yaml
+matrix_jitsi_enable_auth: true
+matrix_jitsi_auth_type: "matrix"
```

Comment:
Instead of making `matrix_user_verification_service_enabled` try to be enabled in a smart way based on these few variables, it's better to be more explicit and just tell people here to add `matrix_user_verification_service_enabled: true` as well.

Then `group_vars/matrix_servers` can just have a default of `matrix_user_verification_service_enabled: false` (overriding the `true` value from the role).

Reply:
We changed that according to your suggestion.

Now someone might set `matrix_jitsi_auth_type: "matrix"`  and forget to enable the user_verification_service. We considered adding a check to the `validate_config.yml` in the matrix-jitsi role to catch the situation. We skipped that since `self_check_matrix_auth.yml` already checks if the user-verification-service is reachable for prosody.